robots.txt

```diff
@@ -1,11 +1,10 @@
 ---
 ---
-{% if site.production == true %}
+{% if site.production %}
 User-agent: *
```

Comment:
Would it, perhaps, make sense to move

`User-agent: *`

outside of the `if` statement?

Reply:
Thanks @pcolmer